README.md

```diff
@@ -112,6 +111,45 @@ query {
   }
 }
 ```
+## Testing Guidelines
+When writing tests, try to follow these guidelines if possible:
+
++ Tests should be as readable as possible and not complex at all. You should understand them by looking at them just once.
++ Local helper functions defined in test files should have functional and easy-to-understand rather than technical names. Meaning, `clickNewUserButton()` is better than `clickElementByTypeAndTestId('button','new-user-button')`.
++ More general use helpers like 'clickElementByTypeAndTestId' can be used within the local helper functions if preferred. The reason for functional naming preference lies in increased readability of tests.
++ Avoid any duplication of helper functions across several files! If using the same functions in several tests (files), there's a tendency to copy-paste the whole file and then rewrite tests. This leads to code duplication of helper functions. Instead, helper functions needed in several locations should be defined in one place should be available globally. Find the matching one by name or create a new one. Avoid creating miscellaneuos file names as it tends to lead to chaos.
++ Current codebase doesn't 100% follow everything stated above but it'd definitely help organising the test helpers accordingly from now on.
+
+![Selection_599](https://user-images.githubusercontent.com/8964422/77221481-6a190d00-6b4a-11ea-88d7-9fc70ce1c982.png)
+
+## Architecture overview
+
+All our architecture decisions are logged in ADRs which you can find [here](docs/adr/adr_template.md).
+Here, is a list of intro tutorials for each technologies / frameworks / languages below.
+
+#### Frontend
+
+- [Typescript](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html)
```

Comment:
We also have quite a lot of stuff written in Pure Javascript. We should probably state that that's still OK.

Reply:
I disagree. I would rather move everything over to Typescript. We should not have a mix.

Reply:
@HaGuesto Most of Katie's code is written in pure JS. Are you proposing that we fully rewrite her code now? Think carefully. As Typescript is a superset of JS, I don't see what the problem is with having JS code here

Reply:
no, I'm not proposing to rewrite it completly. Since typescript is a superset we can leave it if they do not touch it. Maybe just rename the fileendings from .js to .ts if you want. However, for new people I would like to focus on typescript. So, I do not think we should mention js in the CONTRIBUTION guidelines.